### ISSUE TITLE
define selinux::module broken in CentOS 7.3

### ISSUE THREAD

**Issue by edestecd** [voxpupuli/puppet-selinux#142]
In CentOS 7.3 it seems that the .pp file for a selinux module must be the same as the module name.
The whole prefixing scheme of the file name means that the default example no longer works.

**Comment by edestecd** [comment]
This seems to resolve the issue:
```
selinux::module { 'mariadb_galera_cluster':
  content => template("${module_name}/mariadb/galera_cluster.te.erb"),
  prefix  => '',
}
```

Consider making it the default? Empty or no prefix...

**Comment by traylenator** [comment]
See #140

**Comment by bastelfreak** [comment]
Hi @edestecd, the default in the latest release is '', please reopen the issue if you still experience issues.

**Comment by vinzent** [comment]
should be fixed by release 0.7.1 https://github.com/voxpupuli/puppet-selinux/pull/160
